### PR TITLE
Branch 75 - Implement alternative suggested moves

### PR DESCRIPTION
**Pull request by macleanb**
Completed in accordance with issue specification.  The latest commit on main has been merged with this branch (75).  All tests pass.  

75-branch-tests

### CODE REVIEW

**Comment by macleanb** [comment]
This updated version performs better and eliminates the need for an API key.

**Comment by macleanb** [comment]
I just committed another change that removes the (now unnecessary) RapidAPI key import in the backend view.

File: README.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ For aspiring chess players who find personal tutoring cost-prohibitive, CheapChe
 * Create a .env file on the backend and add .env to .gitignore
 * The OPENAI API key will be stored here with one line - export OPEN_AI_KEY="abcdefg..."
 * The Django secret key will also be stored here with one line - DJANGO_SECRET_KEY = '...' (note: leave off 'export')
+* The RapidAPI key will be stored like so: RAPIDAPI_KEY='abcdefg...'
```
Comment (chsbca) [inline README.md:31]: per your last comment on slack, is line 31 still needed?

**Review by chsbca — COMMENTED** [review]
just wanting to check that line on the README.md
if you decide to either remove or keep, feel free to merge the changes. And wow this implementation is badass

**Review by chsbca — APPROVED** [review]

**Review by chsbca — APPROVED** [review]
just wanting to check that line on the README.md
if you decide to either remove or keep, feel free to merge the changes. And wow this implementation is badass

File: README.md (hunk 1 above)
Comment (macleanb) [reply README.md]: Great catch!  I removed line 31 and updated the appearance of the two lines above to improve their styling.

**Pull request merged**